hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java

```diff
@@ -232,7 +232,12 @@
      * invalid.
      */
     SP_AUTO_ASSIGN_DISTRIBUTION_SET_INVALID("hawkbit.server.error.repo.invalidAutoAssignDistributionSet",
-            "The given distribution set for auto-assignment is invalid: it is either incomplete (i.e. mandatory modules are missing) or soft deleted");
+            "The given distribution set for auto-assignment is invalid: it is either incomplete (i.e. mandatory modules are missing) or soft deleted"),
+
+    /**
+     * Error message informing that the current request url contains malicious characters
+     */
+    SP_MALICIOUS_URL_STRING("hawkbit.server.error.rest.maliciousUrl", "The given url contains malicious characters or strings");
```

Comment:
The distinction between characters and strings is irrelevant. Therefore, I would suggest to remove "or strings"